rosenthal/packages/freedesktop.scm

```diff
@@ -46,3 +46,46 @@
                 (sha256
                  (base32
                   "0f72359fzvh6jzri4fd79m34rwm2r55p2ryq4306wrw7xliafzx0")))))))
+
+(define-public xdg-desktop-portal-hyprland
+  (package
+    (name "xdg-desktop-portal-hyprland")
+    (version "0.4.0")
+    (source (origin
+              (method git-fetch)
+              (uri (git-reference
+                     (url "https://github.com/hyprwm/xdg-desktop-portal-hyprland")
+                     (commit (string-append "v" version))))
+              (file-name (git-file-name name version))
+              (sha256
+                (base32
+                  "1dcglmx02j73qbmw9qsacamn8byakyzwknpqjnhsyphixb4crrdg"))))
+    (build-system meson-build-system)
+    (arguments
+      (list
+        #:configure-flags #~(list "-Dsystemd=disabled")))
+    (native-inputs (list cmake pkg-config wayland hyprland-protocols))
```

Comment:
wayland and hyprland-protocols could go inputs.

Reply:
Will do.